open_spiel/games/2048.h

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_2048_H_
+#define OPEN_SPIEL_GAMES_2048_H_
+
+// Implementation of the popular game 2048.
+// https://github.com/gabrielecirulli/2048
+//
+// Some notes about this implementation:
+// - Winning:
+//     The original game continues on even if you reach the coveted 2048 tile,
+//     but in this implementation the game will end so that there's a winning 
+//     end state.
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "open_spiel/spiel.h"
+
+namespace open_spiel {
+namespace two_zero_four_eight {
+
+constexpr int kNumPlayers = 1;
+constexpr int kDefaultRows = 4;
+constexpr int kDefaultColumns = 4;
+
+// The chance tiles that randomly appear on the board after each move
+const std::vector<int> kChanceTiles = {2, 4};
+const int kNoCellAvailableAction = kDefaultRows * kDefaultColumns
+    * kChanceTiles.size();
+
+struct Coordinate {
+  int x, y;
```

Comment:
either use better names (row, column) or add a comment to explain which is representing which

Reply:
Have changed this